packages/react/src/themes/teams/animations/scale.ts

```diff
@@ -3,47 +3,50 @@ import { easeOut, easeIn, easeEasy } from './timingFunctions'
 const scaleAnimations = {
   // Scale in w/ Fade -- Useful for opening modal dialogs
   scaleEnterFast: {
-    keyframe: {
+    keyframe: ({ delta }) => ({
       '0%': {
-        transform: 'scale(.88)',
+        transform: `scale(${delta})`,
         opacity: 0,
       },
       '100%': {
         transform: 'scale(1)',
         opacity: 1,
       },
-    },
+    }),
     duration: '500ms',
     timingFunction: easeOut,
+    keyframeParams: { distance: '.9' },
```

Comment:
let's decide on `distance` or `delta`, it should be the same name.

Reply:
I vote for distance because it's more clear, in my mind.